Formula/mplayer.rb

```diff
@@ -1,10 +1,21 @@
 class Mplayer < Formula
   desc "UNIX movie player"
   homepage "https://mplayerhq.hu/"
-  url "https://mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz"
-  sha256 "650cd55bb3cb44c9b39ce36dac488428559799c5f18d16d98edb2b7256cbbf85"
   license all_of: ["GPL-2.0-only", "GPL-2.0-or-later"]
   revision 1
+  head "svn://svn.mplayerhq.hu/mplayer/trunk"
+
+  stable do
+    url "https://mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz"
+    sha256 "650cd55bb3cb44c9b39ce36dac488428559799c5f18d16d98edb2b7256cbbf85"
+
+    # Backports to fix build with FFmpeg 6:
+    # - r38361 (configure, av_helpers.c: Fix compilation against latest FFmpeg) to fix:
+    #   av_helpers.c:57:34: error: no member named 'decode' in 'struct AVCodec'
+    # - r38411 (ad_spdif.c: Use proper way to set avformat option) to fix:
+    #   libmpcodecs/ad_spdif.c:115:57: error: no member named 'priv_data_size' in 'struct AVOutputFormat'
+    patch :p0, :DATA
```

Comment:
Added bare minimum backports to fix compilation locally, but there are some other commits related to "latest FFmpeg".